app/controllers/frontend_log_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FrontendLogController < ApplicationController
+
+  respond_to :json
+
+  def create
+    if user_fully_authenticated?
+      event, payload = log_params
+      analytics.track_event(event, payload)
+
+      render json: { success: true }, status: :ok
+    else
+      render json: { success: false }, status: :unauthorized
+    end
+
+  rescue ActionController::ParameterMissing => pm
+    render json: { success: false, error_message: pm.to_s }, status: :bad_request
+  end
+
+  private
+
+  def log_params
+    params.require([:event, :payload])
```

Comment:
Should we validate that `payload` is an object? We access properties on it as if it were a hash in the implementation of `Analytics.track_event`, so not sure how gracefully that'd handle a case where we e.g. pass a string value as `payload`.